examples/cube/meson.build

```diff
@@ -0,0 +1,14 @@
+cc = meson.get_compiler('c')
+executable('cube',
+           'cube.c',
+           'vmath.c',
+           dependencies: [
+             dependency('x11'),
+             dependency('gl'),
+             dependency('glu'),
+             cc.find_library('m'),
+             cc.find_library('pthread')
```

Comment:
https://mesonbuild.com/howtox.html#add-math-library-lm-portably
https://mesonbuild.com/FAQ.html#what-is-the-correct-way-to-use-threads-such-as-pthreads

Reply:
thanks, didn't know